packages/cloud-config/definition.yaml

```diff
@@ -1,3 +1,3 @@
 name: cloud-config
 category: system
-version: "0.6.1"
+version: "0.6.2"
```

Comment:
As a reminder, CI now just builds what we bump, so if we don't bump `system/cos` and the squashfs, they won't be tested

Reply:
Gonna bump them too then